fdbclient/NativeAPI.actor.cpp

```diff
@@ -7996,15 +8002,15 @@ ACTOR Future<std::pair<Optional<StorageMetrics>, int>> waitStorageMetrics(
 				return std::make_pair(res, -1);
 			}
 		} catch (Error& e) {
-			TraceEvent(SevDebug, "WaitStorageMetricsError").error(e);
+			TraceEvent(SevDebug, "WaitStorageMetricsError0").error(e);
```

Comment:
Rather than use numbers, I'd propose using a separate name:

```suggestion
			TraceEvent(SevDebug, "WaitStorageMetricsHandleError").error(e);
```